release.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+# Stop at first error
+set -e
+
+if ! command -v gh > /dev/null; then
+    echo "Could not find the GitHub CLI, please install it from https://github.com/cli/cli"
+    exit 1
+fi
+
+# Use version from changelog
+# version=$(head -n1 CHANGELOG.md|cut -d"v" -f2);
+version=$1
+# The target branch, fallback is the default branch
+target=${2:-$(gh repo view --json defaultBranchRef --jq .defaultBranchRef.name)}
+
+if ! [[ $version =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    echo "Invalid version, please enter a valid semantic version"
+    exit 1
+fi
+
+if [[ $(git branch --show-current) != $target ]]; then
+    if ! git switch $target > /dev/null; then
+        echo "Target branch does not exist, please enter a valid branch name"
+        exit 1
+    fi
+fi
+
+echo "Releasing version $version on branch $target";
+
+# Ask for confirmation
+read -r -p "Are you sure? [y/N] " input
+
+case $input in
+    [yY][eE][sS]|[yY])
+        echo "You say Yes"
+        ;;
+    [nN][oO]|[nN])
+        echo "You say No"
+        exit 1
+        ;;
+    *)
+        echo "Invalid input..."
+        exit 1
+        ;;
+esac
+
+# Ask for confirmation
+read -r -p "Create commit and tag? [y/N] " input
+
+case $input in
+    [yY][eE][sS]|[yY])
+        echo "You say Yes"
+
+        # Bump version in info.xml
+        sed -i -E "s|^\t<version>.+</version>|\t<version>$version</version>|" appinfo/info.xml
+
+        # Add changed files to git
+        # git add CHANGELOG.md
```

Comment:
```suggestion
        git add CHANGELOG.md
```
Maybe it makes sense to add this file as it exists and that allows to write the changelog for the app before the release?